base/libgit2/libgit2.jl

```diff
@@ -518,8 +517,7 @@ function merge!(repo::GitRepo;
             filter!(fh->fh.ismerge, fheads)
             if isempty(fheads)
                 throw(GitError(Error.Merge, Error.ERROR,
-                               "There is no fetch reference for this branch."))
-            end
+                               "There is no fetch reference for this branch.")) end
```

Comment:
if it's going to be made a one-liner, should probably use short-circuiting instead of putting the `end` on the same line like this